src/main/java/edu/oakland/helper/admin/TrackData.java

```diff
@@ -145,4 +145,4 @@ public String getStatusMessage() {
   public void setStatusMessage(String statusMessage) {
     this.statusMessage = statusMessage;
   }
-}
```

Comment:
This isn't really a change at all, but I'm not sure how to remove this file from the pull request

Reply:
Just click enter at the end of the file and it should fix itself.